install_config/oab_broker_configuration.adoc

```diff
@@ -90,6 +90,15 @@ secrets: <6>
 <5> See xref:oab-config-broker[Broker Configuration] for details.
 <6> See xref:oab-config-secrets[Secrets Configuration] for details.
 
+[[install-config-oab-running-docker]]
+== Running Docker on each node
+
+During installation, ensure that you are running Docker on all nodes of an OpenShift cluster.
```

Comment:
I took a quick look at the BZ related to this PR and just want to make sure that I'm understanding the reason for running this manual command. In the BZ comment, it says, "This requires a manual step prior to configuring the Automation Broker to use the Partner registry URL." So could this intro sentence could look like:

> Before configuring the Automation Broker, you must run the following command on all nodes of an {product-title} cluster to use the Red Hat Connect partner registry URL:

Note: I'm not sure on the correct naming/capitalization for Red Hat Connect partner registry